### DISCUSSION TITLE
Final Boss: Notorious Liar

### DISCUSSION THREAD

**Discussion by ntseng** [Imaginary-Horizons-Productions/prophets-of-the-labyrinth-Archive#396] [Ideas]
Concept: a boss who gives false information to predicts

**Comment by bobaguardian** [comment]
Oooh I like this idea!

**Comment by doubledh** [comment]
I guess this does present the problem that not having proper predicts isn't something you can exactly build around... other than building enough general mass to dismiss the need for predicts. Might be a good miniboss or normal mob encounter, though

**Reply by ntseng** [reply]
Yeah, that makes a lot of sense. At very least, we should probably demote the fight to an Artifact Guardian.

I think it's also worth considering if we can add more gameplay to the fight (especially since removing the usefulness of predicts is removing gameplay): maybe there are a few enemies, and they all respawn except for the liar? Then the gameplay can be "find the liar".
